### ISSUE TITLE
[Quest][Northrend] Fortunate Misunderstanding

### ISSUE THREAD

**Issue by cbowlick** [dalaranwow/dalaran-wow#3943]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I had just accepted the quest Fortunate Misunderstanding when I was disconnected from the server. I had walked a few paces with the injured oracle beforehand and when I reconnected there were two oracles, the original and the one I had walked with. I tried to abandon the quest to fix it but the out of place oracle stayed put and they both offered me the quest however neither will walk with me. 
**Expected behaviour**: Tell us what should happen instead
I should be able to tell the oracle that I am ready to escort him
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by cbowlick** [comment]
Update: there are now three oracles who all offer me the quest but do not respond to me

**Comment by Rushor** [comment]
only one of them will offer the quest, not all

**Comment by Rushor** [comment]
DB/Creature: Injured Rainspeaker Oracle - Add Npc Flags Gossip & Questgiver on respawn

* closes https://github.com/dalaranwow/dalaran-wow/issues/3943
* commit: eb6ebcadb313af4911e8fc344bbb9ca064952efc

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
> Update: there are now three oracles who all offer me the quest but do not respond to me

With #4523 applied there will be 4.

Regarding:

> DB/Creature: Injured Rainspeaker Oracle - Add Npc Flags Gossip & Questgiver on respawn

Works as intended on PTR, however the gossip option does not show if a player failed to do the quest the first time he tried. Reasons: death, creature death, disconnect, creature too far away (despawn).

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report. This issue was fixed already.
